test/run

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
This isn't necessary for py3. (And can be removed from all other py3 files).

Please split out this commit into a separate PR. This by itself is harmless, we can test it locally and land it fast, and then start rebasing all other PRs onto it, so that the actual change of the test names then doesn't cause so much trouble.